### PR TITLE
Feature/rollup based build system

### PR DESCRIPTION
**Pull request by vinodloha**
Now dist contains UMD, CJS and ES bundles. 

Todo: 
1. Export theme and CSS. 
2. Export other files.
3. Publish script for Nexus.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,6 +133,20 @@
     "styled-components": "^3.4.9",
     "yup": "^0.26.6"
   },
+  "peerDependencies": {
+    "@babel/runtime": ">= 7.1.2",
+    "classnames": ">= 2.2.6",
+    "flexboxgrid": ">= 6.3.1",
+    "formik": ">= 1.3.1",
+    "next": ">= 7.0.1",
```
Comment (ankurkakroo2) [inline package.json]: What is the need for next in this setup ?

Comment (vinodloha) [reply package.json]: The Anchor Component uses Next/Link but we can take it out.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+sudo: false
+before_install:
+- curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.9.4
+- export PATH=$HOME/.yarn/bin:$PATH
+cache:
+  yarn: true
+  directories:
+  - node_modules
+script:
+- yarn run lint
```
Comment (ankurkakroo2) [inline .travis.yml]: why not include lint and test always in the build task

File: package.json (hunk 1 above)
Comment (ankurkakroo2) [reply package.json]: We should take it out. React link should be fine as we are already including react as a peerDependency

**Pull request merged**